src/pages/HomePage.js

```diff
@@ -1,21 +1,54 @@
-import React from 'react';
+import React, { useState, useEffect } from 'react';
 import { FormattedMessage } from 'react-intl';
 import { useSession } from 'hooks';
+import GoogleMapReact from 'google-map-react';
 
 import LogoutButton from 'components/user/LogoutButton';
+import MainLayout from 'components/common/MainLayout';
+
+const defaultCoords = { lat: -26.851311, lng: -65.702984 };
 
 const HomePage = () => {
+  const [locationStatus, setLocationStatus] = useState('');
+  const [lati, setLati] = useState(-26.851311);
+  const [long, setLong] = useState(-65.702984);
   const { user } = useSession();
 
+  useEffect(() => {
+    const success = position => {
```

Comment:
```suggestion
    const success = ({ coords: { latitude, longitude }}) => {
```
You can take advantage of destructuring here